test/tests.zig

```diff
@@ -1882,35 +1884,90 @@ pub fn addCompareOutputTests(
     return cases.step;
 }
 
+/// For stack trace tests, we only test native, because external executors are pretty unreliable at
+/// stack tracing. However, if there's a 32-bit equivalent target which the host can trivially run,
+/// we may as well at least test that!
+fn nativeAndCompatible32bit(b: *std.Build, skip_non_native: bool) []const std.Build.ResolvedTarget {
+    const host = b.graph.host.result;
+    const only_native = (&b.graph.host)[0..1];
+    if (skip_non_native) return only_native;
+    const arch32: std.Target.Cpu.Arch = switch (host.cpu.arch) {
+        .x86_64 => .x86,
+        .aarch64 => .arm,
+        .aarch64_be => .armeb,
+        else => return only_native,
+    };
+    switch (host.os.tag) {
+        .windows => if (arch32.isArm()) return only_native,
```

Comment:
Pretty sure Windows can run 32-bit Arm binaries? (Well, `thumb`, to be exact.)

Reply:
It's not a very important target and it's slowly on its way out, so you don't need to go out of your way to add support for it. This check just stood out to me since I know Windows for Arm64 can run 32-bit binaries.